src/bundle/Resources/public/scss/_content-type-selector.scss

```diff
@@ -23,7 +23,7 @@
     }
 
     &__group-title {
-        color: $ez-color-secondary;
+        color: $ibexa-color-font;
```

Comment:
This variable seems quite strange, what doess `font` mean? Everything is font after all